dev/snippets/lib/configuration.dart

```diff
@@ -67,8 +67,9 @@ class Configuration {
   /// dartdoc.
   Directory get templatesDirectory => Directory(path.join(configDirectory.path, 'templates'));
 
-  /// Gets the skeleton file to use for the given [SnippetType].
-  File getHtmlSkeletonFile(SnippetType type) {
-    return File(path.join(skeletonsDirectory.path, '${getEnumName(type)}.html'));
+  /// Gets the skeleton file to use for the given [SnippetType] and DartPad preference.
+  File getHtmlSkeletonFile(SnippetType type, {bool showDartPad = false}) {
+    final String filename = showDartPad ? 'dartpad.html' : '${getEnumName(type)}.html';
```

Comment:
I know there's only one type right now, but to allow for future types without having to rename the skeleton, maybe rename `dartpad.html` to `dartpad-application.html`?  Then you could change this line to:
```suggestion
    final String filename = '${showDartPad ? 'dartpad-' : ''}${getEnumName(type)}.html';
```

It would also be good to add an assert:
```dart
  assert(!showDartPad || type == SnippetType.application, 'Only application snippets work with dartpad.');
```

Since that only applies in debug mode, A similar check should make the app exit with an error in the `main`.

Reply:
Made all three changes.